### ISSUE TITLE
Edit Question: ua_ruhr_v1.1.0-beta.1/technical-classification/data-dates/data_collection_end

### ISSUE THREAD

**Issue by smohex** [FDM-UARuhr/rdmo-catalog-uaruhr#55]
**Describe the request**  
Adding wording if the collection/creation of data has already ended

**Original setting or content**   

*question text ger*  
"Wann endet die Erhebung bzw. Erstellung der Daten?"

**Proposed setting or content**   

*question text ger*  
"Wann endet/endete die Erhebung bzw. Erstellung der Daten?"
*question text en*
"When does the collection or creation of data end/finished"

**From the following list delete everything that is NOT part of the content of your proposal. Once a subject of change is approved it will be marked accordingly in the list.**  

- [x] question text (en and ger)

**Comment by jmfrenzel** [comment]
https://github.com/rdmorganiser/rdmo-catalog/issues/75

**Comment by ClaraRosalie** [comment]
implemented in upstream rdmorganiser#86
